internal/resource/obclusteroperation/obclusteroperation_task.go

```diff
@@ -157,6 +158,37 @@ func ModifyClusterSpec(m *OBClusterOperationManager) tasktypes.TaskError {
 				obcluster.Spec.Parameters = append(obcluster.Spec.Parameters, apitypes.Parameter{Name: k, Value: v})
 			}
 		}
+	case constants.ClusterOpTypeDeleteOBServers:
```

Comment:
If user pass exact names of observers, why not delete them directly and update replicas of zones?

Reply:
The problem is these are two resource managed by different controllers, if you'd like to do that way, you may add a annotation to a specific observer and modify the replica number of obzone. Adding this operation is mainly for simplicity and to use it in cli tool or dashboard.